### ISSUE TITLE
Feature Request: Make the query area expandable

### ISSUE THREAD

**Issue by eatinasandwich** [Studio3T/robomongo#1508]
Currently the command window starts showing only one line and then expands as your query takes up multiple lines. The expanding stops when it covers about a third of the page and a scrollbar appears. 

It would be super useful if the command window was expandable so you could choose if you wanted to see more of the query or the results (similar to how sql server works). As aggregate queries get longer I find myself copying them into another editor so I can see the whole thing and then back into robo3T to run them.

**Comment by phiferd** [comment]
Yes, please.  This drives me crazy.  Why is there a limit on the query size window at all?
